x/programs/rust/examples/counter-external/src/lib.rs

```diff
@@ -0,0 +1,91 @@
+use wasmlanche_sdk::{public, types::Address, Context, ExternalCallContext, Program};
+
+#[public]
+pub fn inc(_: Context, external: Program, address: Address) {
+    let ctx = ExternalCallContext::new(external, 1_000_000);
+    // TODO: currently need to specify the return type (not good)
+    counter::inc(ctx, address, 1);
+}
+
+#[public]
+pub fn get_value(_: Context, external: Program, address: Address) -> u64 {
+    let ctx = ExternalCallContext::new(external, 1_000_000);
+    counter::get_value(ctx, address)
+}
```

Comment:
The actual functionality that was enabled with this PR

Reply:
👏